.github/workflows/merge-release-to-main.yml

```diff
@@ -0,0 +1,24 @@
+name: Merge Release Into Main
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'release'
+
+jobs:
```

Comment:
what does this do? I thought the main should be the most updated branch? Or it's more like the release branch simply changes the version name?

Reply:
Cool, thanks for the explanation! I think this is worthwhile to add to the contribution doc, which I will do that.